README.md

```diff
@@ -1 +1,6 @@
-# cr-training-mobile-menu
\ No newline at end of file
+# curriculum-cr-training-mobile-menu
+
+
```

Comment:
Kindly make sure to update your Readme File to make it look more professional. Pls use this [template](https://github.com/microverseinc/readme-template).